composer.json

```diff
@@ -12,20 +12,20 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "psr/http-message": "^1.0",
-        "php-http/httplug": "^1.1 || ^2.0",
+        "ext-json": "*",
+        "composer-runtime-api": "^2.0",
+        "php-http/client-common": "^1.9 || ^2.0",
         "php-http/discovery": "^1.0",
         "psr/http-client-implementation": "^1.0",
-        "php-http/client-common": "^1.9 || ^2.0",
         "php-http/message-factory": "^1.0",
-        "composer-runtime-api": "^2.0"
+        "psr/http-message-implementation": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0 || ^9.0",
-        "guzzlehttp/guzzle": "^7",
-        "php-http/mock-client": "^1.0",
         "friendsofphp/php-cs-fixer": "^3.0",
-        "phpstan/phpstan": "^1.2"
+        "php-http/guzzle7-adapter": "^1.0",
```

Comment:
If it doesn't matter. Can you revert back to `guzzlehttp/guzzle`. This is what the docs use and in case this stops working we at least notice this when we run the tests